### PR TITLE
Addresses issues in measure. 

### LINKED ISSUE #13: Implement translation of holidays from honeybee RunPeriod to OpenStudio once the OS SDK supports it
At the time that I am writing this issue, the OpenStudio SDK does not support applying holidays to a simulation.  This should hopefully be addressed in the next OpenStudio release and, once it is addressed, this repo should be updated.

### PR DESCRIPTION
**Pull request by tanushree04**
Closes 
#13 : Includes methods to implement holiday_schedule, and updates schema with schedule_holiday. 
#72 : Includes assertion tests to check whether properties of OpenStudio objects are correctly assigned.

### CODE REVIEW

**Comment by chriswmackey** [comment]
Thanks for doing this @tanushree04 . We should get the tests to pass before we merge.  I can see that [it's failing](https://travis-ci.org/ladybug-tools-in2/energy-model-measure/builds/652235430?utm_source=github_status&utm_medium=notification) seemingly because of all the changes that @nllong has been pushing to the OpenStudio extension gem:
https://github.com/NREL/openstudio-extension-gem/commits/develop

@DavidGoldwasser , do you know that reason for this? Is this rush of changes to the extension gem going to be finished at some point? And, I assume we'll need to upgrade the version of Ruby and OpenStudio we are using once it's done.

File: lib/from_honeybee/geometry/aperture.rb [hunk 1]
```diff
@@ -60,7 +60,11 @@ def to_openstudio(openstudio_model)
       end
       reordered_vertices = OpenStudio.reorderULC(os_vertices)
 
-      os_subsurface = OpenStudio::Model::SubSurface.new(reordered_vertices, openstudio_model)
+      os_subsurface = OpenStudio::Model::SubSurface.new(os_vertices, openstudio_model)
+      os_vertices = os_subsurface.vertices
+      # reorder vertices
+      reordered_vertices = OpenStudio.reorderULC(os_vertices)
+      os_subsurface.setVertices(reordered_vertices)
```
Comment (chriswmackey) [inline lib/from_honeybee/geometry/aperture.rb]: Why did you make this change?  It looks like you are now running the reorder command twice and you have turned a single, efficient line of code into 5. Was there some reason that the previous code was not working? If not, can you change it back for all 4 cases that this code appears?

File: lib/from_honeybee/schedule/ruleset.rb [hunk 2]
```diff
@@ -71,6 +71,15 @@ def to_openstudio(openstudio_model)
...
       end
+
+      # assign holiday schedule
+      if @hash[:holiday_schedule]
+        holiday_schedule = openstudio_model.getScheduleDayByName(@hash[:holiday_schedule])
+        unless holiday_schedule.empty?
+          holiday_schedule_object = holiday_schedule.get
+          os_sch_ruleset.setHolidaySchedule(holiday_schedule_object)
+        end
+      end
```
Comment (chriswmackey) [inline lib/from_honeybee/schedule/ruleset.rb]: Can this be put in a try/except so that it can still work in older OpenStudio versions?

Comment (tanushree04) [reply lib/from_honeybee/schedule/ruleset.rb]: @chriswmackey I added a begin rescue statement that should run if the method exists.

File: lib/from_honeybee/geometry/face.rb [hunk 3]
```diff
@@ -63,10 +63,9 @@ def to_openstudio(openstudio_model)
       end
       reordered_vertices = OpenStudio.reorderULC(os_vertices)
 
-      os_surface = OpenStudio::Model::Surface.new(reordered_vertices, openstudio_model)        
+      os_surface = OpenStudio::Model::Surface.new(reordered_vertices, openstudio_model)
```
Comment (tanushree04) [inline lib/from_honeybee/geometry/face.rb]: @chriswmackey I am not sure why it changed to the earlier reordered_vertices statements. Maybe when I rebased my fork .

File: lib/from_honeybee/schedule/ruleset.rb (hunk 2 above)
Comment (tanushree04) [reply lib/from_honeybee/schedule/ruleset.rb]: @chriswmackey The else statement was not necessart in this case.

**Comment by chriswmackey** [comment]
Thanks @tanushree04 . Good to know. I'll update the travis file before the end of today and, if all of the tests pass, I'll merge this in.

**Pull request merged**